### DISCUSSION TITLE
Add custom_field for Service

### DISCUSSION THREAD

**Discussion by sonbm7112** [netbox-community/netbox#14473] [General]
Hello everyone, I'm new here. Currently, I'd like to add a custom field to the Service model as it doesn't exist yet. What steps should I follow to accomplish this task? Thank you very much!

**Comment by vladimirs-git** [comment]
Creating new [Custom Field](https://demo.netbox.dev/extras/custom-fields/add/) select Object type 'IPAM > Service'

**Reply by sonbm7112** [reply]
Thanks, but what I mean is to add code to the Service's model
